### PR TITLE
readme-CM console integrated instructions

### PR DESCRIPTION
**Pull request by mestew**
Edit readme file with the CM integrated instructions from previous iterations

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,3 +14,47 @@ provided by the bot. You will only need to do this once across all repos using o
 This project has adopted the [Microsoft Open Source Code of Conduct](https://opensource.microsoft.com/codeofconduct/).
 For more information see the [Code of Conduct FAQ](https://opensource.microsoft.com/codeofconduct/faq/) or
 contact [opencode@microsoft.com](mailto:opencode@microsoft.com) with any additional questions or comments.
+
+## Join the Community hub to contribute content
+
+1. Go to the **hub** node in the **Community** workspace.
```
Comment (kmyrup) [inline README.md:20]: Go to the Community hub node in the Community workspace.

File: README.md [hunk 2]
```diff
@@ -14,3 +14,47 @@ provided by the bot. You will only need to do this once across all repos using o
...
+1. Click on **My hub** and you'll be prompted to sign into GitHub. If you don't have an account, you'll be redirected to GitHub where you can create one. A GitHub account is only required to contribute and share content from the **My hub** page.
```
Comment (kmyrup) [inline README.md:21]: Click on **Your hub**.........and share content from the **Your hub** page.

File: README.md [hunk 3]
```diff
@@ -14,3 +14,47 @@ provided by the bot. You will only need to do this once across all repos using o
...
+   ![Join Configuration Manager's Community hub](./media/3555935-join-community-hub.png)
+
+1. After joining, you'll see your membership request is pending. Your account needs approval by the Configuration Manager hub Content Curation team. Approvals are done once a day, so it may take up to one business day for your approval to be granted.
+1. Once you're granted access, you'll get an email from GitHub. Open the link in the email to accept the invitation.
+
+## Contribute content
+
+Once you've accepted the invitation, you can contribute content.
+
+1. Go to **Community** > **hub** > **My hub**.
```
Comment (kmyrup) [inline README.md:33]: Go to **Community** > **Community hub** > **Your hub**.

**Comment by mestew** [comment]
@kmyrup & @silveyy Wow. you guys are all over this. I wasn't even done adding commits. Sending email about this file as well.

File: README.md (hunk 1 above)
Comment (mestew) [reply README.md]: Added in commit https://github.com/microsoft/configmgr-hub/pull/11/commits/0ebfdd1ccb14303527e04f2ac12bca41c1d16c77

File: README.md (hunk 2 above)
Comment (mestew) [reply README.md]: added in commit https://github.com/microsoft/configmgr-hub/pull/11/commits/0ebfdd1ccb14303527e04f2ac12bca41c1d16c77

File: README.md (hunk 3 above)
Comment (mestew) [reply README.md]: resolved in https://github.com/microsoft/configmgr-hub/pull/11/commits/19ef7b0c50cc471f98f8c7f9408cbfa558a57167

**Pull request merged**